python/tk_framework_desktopserver/sgtk_file_dialog.py

```diff
@@ -42,6 +43,13 @@ def __init__(self, multi=False, *args, **kwargs):
         if treeview:
             treeview.setSelectionMode(selection_mode)
 
+        # FIXME: On MacOS the QFileDialog hides all hidden files. Unfortunately /Volumes is hidden.
+        # As a quick hack to unblock our clients, we'll add /Volumes to the sidebar.
+        if sys.platform == "darwin":
+            sidebarUrls = self.sidebarUrls()
+            sidebarUrls.append("file:///Volumes")
+            self.setSidebarUrls(sidebarUrls)
```

Comment:
Awesome workaround!
Only comment is that we should check if that url (or equivalent?) is already there so that we don't duplicate it.